libafl_frida/src/alloc.rs

```diff
@@ -171,7 +171,8 @@ impl Allocator {
             size
         };
         if size > (1 << 30) {
-            panic!("Allocation is too large: 0x{:x}", size);
```

Comment:
Depending on the target this could catch an arithmetic bug though, right? So maybe it'd make sense to make this configurable?

Reply:
hrm. Not a bad idea. Should the threshold size be configurable too?

Reply:
If that's easy to do, why not

Reply:
Done!